content/classes/reading/srp/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Single Responsibility Principle"
+date: 2023-01-23T13:36:52-06:00
+draft: false
+weight: 6
+originalAuthor: Sally Steuterman # to be set by page creator
+originalAuthorGitHub: gildedgardenia # to be set by page creator
+reviewer: Kimberly Horan # to be set by the page reviewer
+reviewerGitHub: codinglikeagirl42 # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+As we wrap up our whirlwind tour of classes, we want you think a bit about how to go about building good classes. Doing so is more of an art than a science, and it will take you lots of practice and time. However, there are a few rules that we’ve pointed out to help guide you. Here’s one more.
+
+From [Wikipedia](https://en.wikipedia.org/wiki/Single_responsibility_principle):
```

Comment:
Suggestion eliminate this link as it is the same as the link below.

Reply:
removed